python/pyspark/rdd.py

```diff
@@ -2406,6 +2406,26 @@ def toLocalIterator(self):
             sock_info = self.ctx._jvm.PythonRDD.toLocalIteratorAndServe(self._jrdd.rdd())
         return _load_from_socket(sock_info, self._jrdd_deserializer)
 
+    def barrier(self):
+        """
+        .. note:: Experimental
+
+        Indicates that Spark must launch the tasks together for the current stage.
+
+        .. versionadded:: 2.4.0
+        """
+        return RDDBarrier(self)
+
+    def isBarrier(self):
```

Comment:
do we have this API in the JVM RDD?

Reply:
In scala RDD there is a `private[spark]` `isBarrier()` function, we don't add this to JavaRDD